bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/update/upgrade.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="goecharger:goe">
+		<instruction-set targetVersion="1">
+			<add-channel id="awattarMaxPrice">
+				<type>unifi:awp</type>
```

Comment:
Why unifi ?
It should be `goecharger:awp`, no?

Reply:
copy/paste Thanks for spotting this!